5.7/test/run

```diff
@@ -110,7 +110,7 @@ function create_container() {
 
 function run_change_password_test() {
   local tmpdir=$(mktemp -d)
-  chmod -R a+rwx "${tmpdir}"
+  mkdir "${tmpdir}/data" && chmod -R a+rwx "${tmpdir}"
```

Comment:
@mhutter I hope last point - Please remove `mkdir "${tmpdir}/data" && ` (also for `5.6`)

`tmpdir` is mounted directly into `/var/lib/mysql/data` so with `data` dir in `tmpdir` mysql data dir is not empty. That's why `5.7` tests are failing.

Reply:
@omron93 ✅